packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view.tsx

```diff
@@ -76,28 +76,31 @@ const OnfidoSdkView = ({
                     />
                 </CSSTransition>
             </div>
-            {is_onfido_disabled && is_onfido_initialized && (
-                <HintBox
-                    className='onfido-container__info-message'
-                    icon='IcInfoBlue'
-                    icon_height={16}
-                    icon_width={16}
-                    message={
-                        <Text as='p' size='xxxs'>
-                            <Localize i18n_default_text='Hit the checkbox above to choose your document.' />
-                        </Text>
-                    }
-                    is_info
+            <section>
+                {is_onfido_disabled && is_onfido_initialized && (
+                    <div className='onfido-container__info-message'>
+                        <HintBox
+                            icon='IcInfoBlue'
+                            icon_height={16}
+                            icon_width={16}
```

Comment:
please remove.. default values

Reply:
It was removed in another PR. Will remove the same here